doc/api/corepack.md

```diff
@@ -12,10 +12,9 @@ added:
 
 > Stability: 1 - Experimental
 
-Documentation for this tool can be found on the [Corepack repository][].
+**Corepack will no longer be distributed with Node.js 25+.**
```

Comment:
nit: 25+ doesn't feel right for docs
```suggestion
**Starting with Node.js v25, Corepack will no longer be distributed.**
```

Reply:
Won't this cause confusion where readers might think Corepack distribution is going to be stopped everywhere, if they miss reading call to action which is not in bold?

We want to say the Corepack distribution is going to be stopped only on Node.js.

Reply:
How about this edit? This just changes the 25+ part
```suggestion
**Corepack will no longer be distributed starting with Node.js v25.**
```